packages/contracts-bedrock/contracts/test/MintManager.t.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { CommonTest } from "./CommonTest.t.sol";
+import { MintManager } from "../governance/MintManager.sol";
+import { GovernanceToken } from "../governance/GovernanceToken.sol";
+
+contract MintManager_Test is CommonTest {
+    address constant owner = address(0x1234);
+    address constant rando = address(0x5678);
+    GovernanceToken internal gov;
+    MintManager internal manager;
+
+    function setUp() external {
+        vm.prank(owner);
```

Comment:
Could just use `alice` here, and then delete the state var.

Reply:
I prefer having the explicit variable name